ChakraCore Samples/Hello World/Linux_OSX/Makefile

```diff
@@ -9,13 +9,16 @@ CC=g++
 LIBRARY_PATH=../BuildLinux/$(BUILD_TYPE)/lib
 LDIR=$(LIBRARY_PATH)/../pal/src/libChakra.Pal.a \
 	$(LIBRARY_PATH)/Common/Core/libChakra.Common.Core.a \
-	$(LIBRARY_PATH)/Jsrt/libChakra.Jsrt.a
+	$(LIBRARY_PATH)/Jsrt/libChakra.Jsrt.a \
+	/usr/local/opt/icu4c/lib/libicudata.a \
+	/usr/local/opt/icu4c/lib/libicuuc.a \
+	/usr/local/opt/icu4c/lib/libicui18n.a 
```

Comment:
@juliusl Should we add those lines only for Mac or does it work also for other platforms ?